scripts/Get-Tags.ps1

```diff
@@ -0,0 +1,17 @@
+param(
```

Comment:
@natemcmaster FYI this was inspired from (https://github.com/aspnet/aspnet-docker/blob/master/scripts/GenerateTags.ps1)